hibike/hibike_process.py

```diff
@@ -6,15 +6,22 @@
 import glob
 import serial
 import os
+from platform import system
 
 __all__ = ["hibike_process"]
 
 
 uid_to_index = {}
 
 def hibike_process(badThingsQueue, stateQueue, pipeFromChild):
-
-    ports = glob.glob("/dev/ttyACM*") + glob.glob("/dev/ttyUSB*")
+    
+    # So that it's compatible with OS X Sierra
+    # Note: If you are running OS X Sierra, do not access the directory through vagrant ssh
+    # Instead access it through Volumes/vagrant/PieCentral
+    if system() == "Darwin":
```

Comment:
I think that all reasonably modern versions of OSX/macOS all use tty.usbmodem*, but yeah I don't really see a reason not to just combine all of the globs onto one line.